.github/workflows/mirror-docker-image.yml

```diff
@@ -0,0 +1,37 @@
+name: mirror-docker-image
+
+on:
+  workflow_dispatch:
+    inputs:
+      image:
+        type: string
+        description: |
+          oras-compatible image ref to mirror (ex. 'docker.io/library/busybox')
+      version:
+        type: string
+        description: |
+          Version of the source image to mirror (ex. '@<digest>', ':<tag>')
+      output-image:
+        description: |
+          Image path under 'wasmcloud/mirrors' to use (ex. 'a' -> 'wasmcloud/mirrors/a:<tag>', 'tools/b' -> 'wasmcloud/mirrors/tools/b:<tag>')
+        optional: true
+
+permissions: {}
+
+jobs:
+  mirror-docker-image:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+    steps:
+      - uses: oras-project/setup-oras@ca28077386065e263c03428f4ae0c09024817c93
+      - name: Docker login
+        shell: bash
+        run: |
+          echo ${{ secrets.GITHUB_TOKEN }} | oras login ghcr.io --username ${{ github.repository_owner }} --password-stdin
+      - name: Copy image
+        shell: bash
+        run: |
+          oras cp \
+          ${{ inputs.image }}${{ inputs.version }} \
+          ghcr.io/${{ github.repository_owner }}/mirrors/${{ inputs.output-image }}${{ inputs.version }}
```

Comment:
You will need the lowercased version of `github.repository_owner`: https://github.com/wasmCloud/wasmCloud/blob/7e26ac135ff6e6f9678f21e44e9631734311c264/.github/workflows/wasmcloud-chart.yml#L81-L83

OCI doesn't support uppercase letters in the identifier.

Reply:
Ahh thanks -- will add this in!

- [ ] Add in lowercasing of repo owner